Dockerfile

```diff
@@ -1,8 +1,26 @@
+# For more information, please refer to https://aka.ms/vscode-docker-python
 FROM python:3.9-slim-bullseye as base
 
-WORKDIR /service
+# Keeps Python from generating .pyc files in the container
+ENV PYTHONDONTWRITEBYTECODE=1
 
-RUN pip install poetry
-COPY . /service
+# Turns off buffering for easier container logging
+ENV PYTHONUNBUFFERED=1
 
-CMD poetry install && poetry run python ./bot
+# Install pip requirements
+COPY requirements.txt .
+COPY requirements-dev.txt .
+RUN python -m pip install -r requirements-dev.txt
+
+RUN python -m pip install https://github.com/Pycord-Development/pycord/archive/master.zip
```

Comment:
It may be better to move this to the requirements.txt, and we should definitely pin this to a specific commit to avoid having the package breaking between builds.

This should work in the requirements.txt
```
-e git+https://github.com/Pycord-Development/pycord@1df0afa#egg=py-cord

```

Reply:
I added a git install to the dockerfile to support this, but will remove when a release version of py-cord is available